decentralized_payrole/README.md

```
@@ -0,0 +1,268 @@
+```
+
+### 2. Add Supported Tokens
+
+;; Add SIP-010 compliant tokens (STX is native and doesn't need to be added)
+;; Example with a real SIP-010 token:
+(contract-call? .pay_role add-supported-token 'SP32AEEF6WW5Y0NMJ1S8SBSZDAY8R5J32NBZFPKKZ.wrapped-stx)
+
+;; Add other SIP-010 tokens as needed
+(contract-call? .pay_role add-supported-token 'TOKEN_CONTRACT_ADDRESS)
+
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)